### PR TITLE
Have LOCAL_PORTS override anything obtained from docker

### PR DESCRIPTION
**Pull request by jared-mackey**

### CODE REVIEW

File: smallprox/mapper.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
 logger = logging.getLogger('small-prox')
 
+async def update_local_overrides(config: dict):
```
Comment (nhumrich) [inline smallprox/mapper.py]: This doesnt need to be `async`

Comment (nhumrich) [inline smallprox/mapper.py]: This doesnt need to be `async`

**Review by nhumrich — CHANGES_REQUESTED** [review]

File: smallprox/mapper.py (hunk 1 above)
Comment (jared-mackey) [reply smallprox/mapper.py]: resolved

**Review by nhumrich — APPROVED** [review]
🐙

**Pull request merged**